ui/org.eclipse.pde.ui/src/org/eclipse/pde/ui/launcher/PluginsTab.java

```diff
@@ -217,16 +217,9 @@ public Image getImage() {
 		return fImage;
 	}
 
-	/**
-	 * Validates the tab.  If the feature option is chosen, and the workspace is not correctly set up,
-	 * the error message is set.
-	 *
-	 * @see org.eclipse.pde.ui.launcher.AbstractLauncherTab#validateTab()
-	 */
```

Comment:
instead of removing the doc, could you modify it to reflect what the public API is doing?

Reply:
the behavior was changed back in 2010 (https://github.com/eclipse-pde/eclipse.pde/commit/b787978d3202383ca77c1ad4c37a36b5863997f5), so better not.

Reply:
I agree, so I am not advocating to retain the doc as is. Instead, modify the doc to reflect the new behaviour. (premise: when API is modified, doc is adjusted too to match the new behaviour)

Reply:
A JavaDoc here would only be outdated the next time someone decides to change the implementation to actually do something. I think the default one is ok:

![image](https://github.com/eclipse-pde/eclipse.pde/assets/2205684/f2b662b9-9fca-47f7-8b4f-824ea1eb52f8)